ckanext/canada/templates/public/package/snippets/resources_list.html

```diff
@@ -0,0 +1,15 @@
+{% ckan_extends %}
+
+{% block resource_list_title %}
+  <h2>{{ _('Data and Resources') }}
+  <div class="pull-right mrgn-bttm-md">
+    {% if 'fgp_viewer' in pkg.get('display_flags', []) %}
+      {% set openmap_uri = 'openmap' %}
+      {% if h.lang() == 'fr' %}
+        {% set openmap_uri = 'carteouverte' %}
+      {% endif %}
+      <a href="{{ h.adv_search_url() }}/{{ openmap_uri }}/{{ pkg.name }}" class="btn btn-success" target="_blank">
```

Comment:
you could save some repetition and business-logic-in-templates here by creating a helper for the openmap_uri instead

Reply:
good idea. I have added a helper `fgp_viewer_url` and remove old helper `fgp_url` which was not used by any templates